geometria2.tex

```diff
@@ -45,6 +45,8 @@ \section{Esercizi svolti}
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \rule{0.3\linewidth}{0.25pt}
 \scriptsize\\
-\href{mailto:marco.donadoni@mail.polimi.it}{M. Donadoni}, \href{mailto:edoardo.morassutto@mail.polimi.it}{E. Morassutto},  Politecnico di Milano, A.A. 2016/17
+\href{mailto:marco.donadoni@mail.polimi.it}{M. Donadoni}, \href{mailto:edoardo.morassutto@mail.polimi.it}{E. Morassutto},
+\href{mailto:alessandro.fulgini@mail.polimi,it}{Alessandro Fulgini} \\
```

Comment:
Hey! Hey! Hey! 

`N. Cognome` è la forma corretta di inserire il proprio nome in questo foglio!